### PR TITLE
34563 Map in property listing

### PR DESCRIPTION
**Pull request by yeneastgate**
related to https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/516

changed log :
Add checkbox to show map in estate list

### CODE REVIEW

**Review by andernath — COMMENTED** [review]
Finished review and I think we need a manual merge here. Please check @yeneastgate :)

File: plugin/Installer/DatabaseChanges.php [hunk 1]
```diff
@@ -44,7 +44,7 @@
 class DatabaseChanges implements DatabaseChangesInterface
 {
 	/** @var int */
-	const MAX_VERSION = 38;
+	const MAX_VERSION = 39;
```
[the remark this answers is not in the document]
Comment (yeneastgate) [reply plugin/Installer/DatabaseChanges.php:47]: @andernath Yes, I will resolve conflicts and merge this branch into master

[the remark this answers is not in the document]
Comment (andernath) [reply plugin/Installer/DatabaseChanges.php]: @yeneastgate for a valid test we need to **merge master into this branch**

**Review by andernath — APPROVED** [review]
I will start testing

**Pull request merged**